src/library/trackset/setlogfeature.cpp

```diff
@@ -92,6 +92,9 @@ void SetlogFeature::bindLibraryWidget(
             this,
             &SetlogFeature::slotPlayingTrackChanged);
     m_libraryWidget = libraryWidget;
+    connect(m_libraryWidget, &WLibrary::destroyed, [this] {
```

Comment:
You have basically reimplemented QPinter here.

Reply:
I second that. Use QPointer instead.